single-location.php

```diff
@@ -2,46 +2,60 @@
 get_header();
 the_post();
 
+$address = ( isset( $post->ucf_location_address ) ) ? '<h2 class="h6 font-weight-normal">' . $post->ucf_location_address . '</h2>' : "";
 $lat_lng = $post->meta['ucf_location_lat_lng'];
 $lat_lng_string = ( $lat_lng ) ? $lat_lng["ucf_location_lat"] . "," . $lat_lng["ucf_location_lng"] : "";
 $events = ( isset( $post->meta['events_markup'] ) && ! empty( $post->meta['events_markup'] ) ) ? $post->meta['events_markup'] : null;
+$featured_image = wp_get_attachment_url( get_post_thumbnail_id($post->ID) );
+$location_fallback_image = "";
+
+if( ! empty( get_theme_mod_or_default( 'location_fallback_image' ) ) ) {
+	$location_fallback_image = '<div class="col-md-6"><img src="' . get_theme_mod_or_default( 'location_fallback_image' ) . '" alt="" class="img-fluid"></div>';
+}
 ?>
 
-<div class="container mt-4 mb-4 mb-sm-5 pb-md-3">
+<div class="container mt-5 pt-4 mb-4 mb-sm-5 pb-md-3">
 	<div class="row">
-		<div class="col-md-8 col-xl-8">
-
-			<div class="d-block d-md-none">
-				<?php echo get_location_html(); ?>
+		<div class="col-lg-7">
+			<div class="row mb-3">
+					<?php if( $featured_image ) : ?>
+						<div class="col-md-6">
+							<img src="<?php echo $featured_image ?>" alt="" class="img-fluid mb-3">
+						</div>
+					<?php else : ?>
+						<?php echo $location_fallback_image; ?>
+					<?php endif; ?>
+				<div class="col-md-6">
+					<?php echo get_location_html(); ?>
+				</div>
 			</div>
 
-		<?php if( ! empty ( $lat_lng_string ) ) : ?>
-			<iframe width="100%" height="300" frameborder="0" allowfullscreen=""
-				src="https://www.google.com/maps/embed/v1/view?key=<?php echo htmlentities( get_theme_mod_or_default( 'google_map_key' ) ); ?>&amp;center=<?php echo $lat_lng_string; ?>&amp;maptype=satellite&amp;zoom=18"></iframe>
-		<?php endif; ?>
-
-			<a href="https://map.ucf.edu/?show=<?php echo $post->ucf_location_id; ?>" class="text-right text-uppercase d-block mb-3 pr-2">View on Map</a>
+			<?php if( ! empty( the_content() ) ) : ?>
+				<p><?php the_content(); ?></p>
```

Comment:
Don't wrap `the_content()` in `<p>` tags here; otherwise it's possible for nested `<p>` tags to be rendered.  Use `<div>` instead if you need to ensure the post content is wrapped in something.